src/customer/customer-strategy-action-creator.spec.ts

```diff
@@ -42,7 +43,8 @@ describe('CustomerStrategyActionCreator', () => {
 
         state = getCheckoutStoreState();
         store = createCheckoutStore(state);
-        registry = createCustomerStrategyRegistry(store, createRequestSender());
+        locale = 'en';
+        registry = createCustomerStrategyRegistry(store, createRequestSender(), locale);
```

Comment:
Same here, instead of assigning to a temporary variable, you could directly pass it to the function